backend/compact-connect/common_constructs/webacl.py

```diff
@@ -66,14 +66,14 @@ def __init__(
                 raise RuntimeError('CLOUDFRONT scoped WebACLs must be in the "us-east-1" region to have'
                                    ' logging enabled')
 
-            log_group = LogGroup(
+            self.log_group = LogGroup(
```

Comment:
I assume this was a bug from a prior PR?

Reply:
Hm, I think this was drafted in from when I cleaned up a huge git history mess (fallout from converting from per-iteration to per-feature PRs). I'll back that out.

Reply:
On second thought, this change is self-contained and beneficial. What it is doing is making the log group available as an attribute, so that it can be included in Query definitions by the consuming stacks, which we will leverage separately. I'll leave this in.

Reply:
Got it!